src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -855,6 +855,12 @@ public CaseDataBuilder atStateProceedsOfflineAfterClaimNotified() {
         return this;
     }
 
+    public CaseDataBuilder atStateProceedsOfflineAfterClaimDetailsNotified() {
```

Comment:
Can we utilise this method in here:
https://github.com/hmcts/civil-service/blob/8786a74f907317cde01008c25be67261b29c9287/src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java#L477

Something like this:


                case TAKEN_OFFLINE_AFTER_CLAIM_DETAILS_NOTIFIED:
                return atStateProceedsOfflineAfterClaimDetailsNotified();